webapp/src/components/EspOta.vue

```diff
@@ -0,0 +1,139 @@
+<template>
+  <v-card>
+    <v-card-text>
+        <v-row justify="center" dense>
+            <v-col cols="10">
+            <div class="ma-auto">
+                Select an image from your PC to use. Please note that this process can take up to 5 minutes to complete!
+            </div>
+            </v-col>
+        </v-row>
+        <v-row justify="center" dense>
+            <v-col cols="10">
+            <div class="ma-auto">Current Version: {{ oldFwVersion }}</div>
+            </v-col>
+        </v-row>
+        <v-row justify="center">
+            <v-col cols="10">
+            <v-file-input
+                v-model="file"
+                label="File input"
+                outlined
+                dense
+                :show-size="1000"
+            ></v-file-input>
+            </v-col>
+        </v-row>
+        <v-row justify="center" dense>
+            <v-col cols="2">
+            <v-btn
+                :loading="uploading"
+                @click="start()"
+            >Start</v-btn>
+            </v-col>
+        </v-row>
+        <v-row justify="center">
+            <v-alert
+            v-model="alert"
+            dense
+            outlined
+            type="error"
+            transition="scale-transition"
+            dismissible
+            >{{ status }}</v-alert
+            >
```

Comment:
Please use consistent indentation (2 spaces in JavaScript and HTML files, 4 spaces in C/C++ files).